src/go/plugin/go.d/collector/mysql/charts.go

```diff
@@ -364,6 +367,42 @@ var (
 			{ID: "innodb_log_writes", Name: "writes", Algo: module.Incremental, Mul: -1},
 		},
 	}
+	chartInnoDBLogOccupancy = module.Chart{
+		ID:       "innodb_log_occupancy",
+		Title:    "InnoDB Redo Log Occupancy",
+		Units:    "percentage",
+		Fam:      "innodb",
+		Ctx:      "mysql.innodb_log_occupancy",
+		Type:     module.Area,
+		Priority: prioInnoDBLog,
+		Dims: module.Dims{
+			{ID: "innodb_log_occupancy", Name: "occupancy", Algo: module.Absolute, Div: 1000},
+		},
+	}
+	chartInnoDBLogProgress = module.Chart{
+		ID:       "innodb_log_progress",
+		Title:    "InnoDB Log Sequence Progress",
+		Units:    "MiB/s",
+		Fam:      "innodb",
+		Ctx:      "mysql.innodb_log_progress",
+		Type:     module.Area,
+		Priority: prioInnoDBLog,
+		Dims: module.Dims{
+			{ID: "innodb_log_sequence_number", Name: "sequence number", Algo: module.Incremental, Div: 1024 * 1024},
+			{ID: "innodb_last_checkpoint_at", Name: "last checkpoint", Algo: module.Incremental, Mul: -1, Div: 1024 * 1024},
+		},
```

Comment:
1. Keep the units in bytes per second (B/s) instead of converting to MiB/s.

2. Can you explain how `sequence number` and `last checkpoint` can be represented in bytes per second? I need to understand what these values actually measure. We might need to rename the metrics or split them into separate charts.